### PR TITLE
bump(github.com/openshift/source-to-image): 643ad40f6a1df322e7b41e861ac6623f524f84bf

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@php-coder ptal
[test]

**Comment by bparees** [comment]
[testextended][extended:core(builds)]

File: vendor/github.com/openshift/source-to-image/pkg/docker/test/client.go [hunk 1]
```diff
@@ -0,0 +1,146 @@
+package test
```
Comment (php-coder) [inline vendor/github.com/openshift/source-to-image/pkg/docker/test/client.go]: Why this file wasn't imported before and now it's completely new? Do we really need it?

Comment (liggitt) [inline vendor/github.com/openshift/source-to-image/pkg/docker/test/client.go]: It got removed yesterday with a real godep save.

**Comment by php-coder** [comment]
>pkg/build/builder/source.go:154: multiple-value "github.com/openshift/origin/vendor/github.com/openshift/source-to-image/pkg/scm/git".New().ValidCloneSpec() in single-value context

@bparees Looks like we also need to update origin code.

File: vendor/github.com/openshift/source-to-image/pkg/docker/test/client.go (hunk 1 above)
Comment (liggitt) [inline vendor/github.com/openshift/source-to-image/pkg/docker/test/client.go]: probably so, looks like that script is already trying to exclude all test-only files and packages

File: vendor/github.com/openshift/source-to-image/LICENSE [hunk 2]
```diff
@@ -1,202 +0,0 @@
-Apache License
```
Comment (liggitt) [inline vendor/github.com/openshift/source-to-image/LICENSE]: also looks like it's removing the LICENSE godeps includes

File: vendor/github.com/openshift/source-to-image/pkg/build/ignore/doc.go [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// implements an interface around providing ignore file capabillities like seen in .dockerignore or .gitignore
```
Comment (liggitt) [inline vendor/github.com/openshift/source-to-image/pkg/build/ignore/doc.go]: exclude this package as well, no imported package depends on it

**Comment by stevekuznetsov** [comment]
[test comment please ignore]
#1234
https://github.com/openshift/origin/issues/1234
https://ci.openshift.redhat.com/jenkins/job/test_pr_origin/7565/
https://ci.openshift.redhat.com/jenkins/job/test_pr_origin_extended/407/

**Comment by bparees** [comment]
@php-coder i'm closing this since you want to do another bump for your additional changes anyway.  you can fix up the rebase script before you do your bump.

**Comment by php-coder** [comment]
@bparees 
PR for fixing `pkg/build/ignore/doc.go`: https://github.com/openshift/source-to-image/pull/562
PR for fixing the script: https://github.com/openshift/source-to-image/pull/563
PR for a new bump: https://github.com/openshift/origin/pull/10277

**Pull request closed without merging**